var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -205,5 +208,9 @@ def cmake_args(self):
         elif 'blas=mkl' in spec:
             args.extend([
                 '-DHydrogen_USE_MKL:BOOL=%s' % ('blas=mkl' in spec)])
+        elif 'blas=accelerate' in spec:
+            args.extend([
+                '-DHydrogen_USE_ACCELERATE:BOOL=%s' % (
+                    'blas=accelerate' in spec)])
```

Comment:
@bvanessen This could be simplified slightly, since we only got here if `'blas=accelerate' in spec == True`, so you could add `'-DHydrogen_USE_ACCELERATE:BOOL=True`.